INFO:__main__:Fitting model at time: 2021-09-09 09:03:38.846489
Traceback (most recent call last):
  File "train_model.py", line 171, in <module>
    model, train_loss, test_loss = train_model(
  File "train_model.py", line 113, in train_model
    out = model(batch.to(DEVICE))
  File "/zhome/2b/7/117471/Thesis/venv-thesis/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/zhome/2b/7/117471/Thesis/src/models/models.py", line 572, in forward
    out, _ = self.lstm(lstm_inputs)
  File "/zhome/2b/7/117471/Thesis/venv-thesis/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051, in _call_impl
    return forward_call(*input, **kwargs)
  File "/zhome/2b/7/117471/Thesis/venv-thesis/lib/python3.8/site-packages/torch/nn/modules/rnn.py", line 679, in forward
    result = _VF.lstm(input, hx, self._flat_weights, self.bias, self.num_layers,
KeyboardInterrupt
